stable/stolon/values.yaml

```diff
@@ -77,6 +91,13 @@ keeper:
   tolerations: []
   volumes: []
   volumeMounts: []
+  hooks:
+    failKeeper:
+      enabled: false
+  podDisruptionBudget:
+    enabled: false
```

Comment:
I would make it simpler. The `enabled` flag is unnecessary. Here's an example:

https://github.com/helm/charts/blob/master/stable/keycloak/templates/poddisruptionbudget.yaml#L16
https://github.com/helm/charts/blob/master/stable/keycloak/values.yaml#L135-L137

Reply:
Thanks for example :)
removed `podDisruptionBudget.enabled`